From d4bb2088c16e86e79e878f7cf0f7fa4c7d6acf8b Mon Sep 17 00:00:00 2001
From: Kleis Auke Wolthuizen <github@kleisauke.nl>
Date: Wed, 18 Sep 2019 15:40:00 +0200
Subject: [PATCH 05/18] Network libs are not available on Emscripten

Upstream-Status: Inappropriate [Wasm-specific]
GLib will not integrate any Wasm-specific patches until the toolchain
supports calling functions through differently typed function pointers,
see: https://gitlab.gnome.org/GNOME/glib/-/issues/3596

(cherry picked from commit c6a63fcd426f282e31aefb33a6011e98dd0405d1)
---
 gio/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/meson.build b/gio/meson.build
index 02a5d04af..aff84682c 100644
--- a/gio/meson.build
+++ b/gio/meson.build
@@ -39,7 +39,7 @@ endif
 
 network_libs = [ ]
 network_args = [ ]
-if host_system != 'windows'
+if host_system not in ['windows', 'emscripten']
   # res_query()
   res_query_test = '''#include <resolv.h>
                       int main (int argc, char ** argv) {
